doc/nrf/config_and_build/dfu/index.rst

```diff
@@ -103,8 +103,13 @@ Device-specific DFU guides
 
 See the following pages for device-specific guides related to DFU:
 
-* :ref:`qspi_xip` - for the nRF5340 SoC
-* :ref:`ug_nrf70_fw_patch_update` - for nRF70 Series devices
+* :ref:`Developing with nRF52 Series <ug_nrf52_developing_ble_fota>` - for how to do FOTA updates with nRF52 Series devices.
+
+* :ref:`Developing with nRF5340 DK <ug_nrf53_developing_ble_fota>` - for how to FOTA updates with the nRF5340 SoC.
```

Comment:
Nit: nRF53 docs describe both FOTA and Serial Recovery(Which technically is not Over the Air).

Not  important, as people will likely look there for any kind of DFU. The hotlink between the pages are the important part after all